ocean_lib/data_provider/test/test_base.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright 2022 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+from ocean_lib.data_provider.base import DataServiceProviderBase
+
+
+def test_sanitize_content_disposition():
```

Comment:
please add tests for _get_file_name as well, not only for _validate_content_disposition